src/lib/y2network/interface_config_builders/bonding.rb

```diff
@@ -25,46 +25,28 @@ module Y2Network
   module InterfaceConfigBuilders
     class Bonding < InterfaceConfigBuilder
       include Yast::Logger
+      extend Forwardable
 
       def initialize(config: nil)
         super(type: InterfaceType::BONDING, config: config)
-
-        # fill mandatory bond option
-        @slaves = []
       end
 
       # @return [Array<Interface>] list of interfaces usable for the bond device
       def bondable_interfaces
         interfaces.all.select { |i| bondable?(i) }
       end
 
-      # @return [String] options for bonding
-      attr_writer :bond_options
-      # current options for bonding
-      def bond_options
-        return @bond_options if @bond_options
-
-        @bond_options = Yast::LanItems.bond_option
-        if @bond_options.nil? || @bond_options.empty?
-          @bond_options = @config["BONDING_MODULE_OPTS"]
-        end
-        @bond_options
-      end
+      def_delegators :connection_config,
+        :slaves, :slaves=
 
-      def slaves
-        @slaves
+      # @param value [String] options for bonding
+      def bond_options=(value)
+        connection_config.options = value
       end
 
-      def slaves=(value)
-        @slaves = value
-      end
-
-      # (see Y2Network::InterfaceConfigBuilder#save)
-      def save
-        super
-
-        Yast::LanItems.bond_slaves = @slaves
-        Yast::LanItems.bond_option = bond_options
+      # current options for bonding
```

Comment:
np: Please, annotate the type. Given that it is called `options`, somebody could expect an array.